### ISSUE TITLE
Fullscreen toggling on certain keys

### ISSUE THREAD

**Issue by Proximyst** [NillerMedDild/Enigmatica2Expert#1255]
**Modpack version**
1.64d

**Can the issue be reproduced?**
Yes

**Logs**
i3 provides nothing and I haven't heard anything from my friend using bspwm

**Issue**
I use the i3 window manager with the compton compositor on Arch Linux, and I play with a friend using the bspwm window manager on Void Linux. This then means it's not distro or WM dependent. We both only have $mod+f set to toggle fullscreen (which any app can change as it likes, e.g. when watching YT in our browsers). When we press a certain key which is not within the standard range of (text; tab, caps, etc. work fine) keys to press (for me A-Z 0-9 `,.-'` and all shift versions of these), it changes fullscreen mode. This is not an issue within vanilla 1.12.2 or other 1.12.2 modpacks and leads me to believe some mod in this pack changes how it works. This is rather annoying given how many times it changes when we wanna check Discord, change music, or even just ask a question then have `?` change fullscreen and erase the chatbox.

If there is some mod which "helps" in controlling the fullscreen mode and whatnot of the game, we'd love to hear what mod this would be.

**Comment by Proximyst** [comment]
After trying to look thru some mods, we found Fullscreen Windowed. Removing it solved all of the issues. Might it be reconsidered being put in the pack, or at least have some other mod regulating it to not apply to non-DE environments?

**Comment by NillerMedDild** [comment]
I'm sorry you encountered this issue, but seeing as it's extremely rare, I'm not sure it's enough reason to remove the mod. I'll think it over though.

**Comment by Frikkle** [comment]
@Niller it might be an idea to make a note about this in the (installation) docs. There might be other mods that are optional for the functioning of the pack which cause issues on certain setups. They could be taken out manually by people experiencing them.

On the other hand: people reading the docs are the same people that would search the issue tracker and find this post. Unfortunately most people would do neither :p

**Comment by Proximyst** [comment]
I +1 @Frikkle on this. Those who happen to have issues with this are most likely using certain WMs (note: Windows (and macOS?) do not support any WMs which aren't their own) are also likely using some form for a curse downloader given Twitch Launcher is not a thing on Linux and BSD. These people are then also much more likely to read the page of the modpack and/or here.

**Comment by Frikkle** [comment]
So it's good that this got documented here.

**Comment by NillerMedDild** [comment]
... for the second time.

**Comment by Dark-Matter-001** [comment]
While playing in windowed mode, the calculator button on my keyboard causes Minecraft to lock itself into full screen as well; with the only way to disable being shut down the game and edit the options.txt to set fullscreen:false.  It never happened back in 1.58 and looking at the twitch launcher backup of that install, FullscreenWindowed never existed in that version.  Very annoying since using the calculator happens a lot while playing.

After seeing this issue and removing FullscreenWindowed-1.12-1.6.0.jar, I can once again hit/use my calculator button without messing up the game into requiring manual config edits and reloads.

**Comment by grecolons** [comment]
Today playing version 1.67 I changed they key binding for the fullscreen. I had it on NONE and when I used some symbols like '<' the fullscreen was toggled. To solve that I changed the key ingame for crtl+f. The game crashed and no matter what I did when the launcher finished it would crash again. 
Someone in reddit told me to go to options.txt and change from there the key binding to none again, that worked and I'm playing again.
